evalai/challenges.py

```diff
@@ -234,4 +235,52 @@ def submit(ctx, file):
     make_submission(ctx.challenge_id, ctx.phase_id, file, submission_metadata)
 
 
+@phase.command(context_settings={"ignore_unknown_options": True})
+@click.pass_obj
+@click.option("--file", type=click.STRING, required=True, help="File path to the submission file")
+def upload_submission(ctx, file):
```

Comment:
@KhalidRmb Isn't it possible to use the previous command only for creating submissions. Having two different commands for the same action seems a bit weird to me.

You can add a parameter `-l` or `--large` and decide on the basis of that.
cc: @Ram81 @deshraj

Reply:
Makes sense..Will add the change.

Reply:
Done @RishabhJain2018